draft/2022-02-16-this-week-in-rust.md

```diff
@@ -26,6 +26,8 @@ If you find any errors in this week's issue, [please submit a PR](https://github
 
 ### Rust Walkthroughs
 
+* [video] [5 Tips for Rust Beginners](https://youtu.be/zdT3bUljGQw)
```

Comment:
We ask that, to be consistent, people use the youtube.com link, rather than the shortened youtu.be. Could you update this link?

Reply:
Thanks @nellshamrell, I've changed it